tests/simple.sh

```diff
@@ -0,0 +1,132 @@
+#/bin/bash
+# A quick port of the travis tests to bash, requires root
+# TODO
+# * look into using a framework..
+# * why --dry-run fails?
+# * why --experimental fails?
+# * https://github.com/sosreport/sos/issues/1921
+# * make it better validate archives and contents
+
+PYTHON=${1:-/usr/bin/python3}
+SOSPATH=${2:-./sosreport}
+
+NUMOFFAILURES=0
+
+run_expecting_sucess () {
+    FAIL=false
+    # Make sure clean
+    rm -f stderr stdout
+
+    echo "######### RUNNING $1 #########"
+    $PYTHON $SOSPATH $1 2> stderr 1> stdout
+
+    if [ $? -eq 0 ]; then
+      echo "### Success"
+    else
+      echo "!!! FAILED !!!"
+      FAIL=true
+    fi
+
+    if [ -s stderr ]; then
+       FAIL=true
+       echo "!!! FAILED !!!"
+       echo "### start stderr"
+       cat stderr
+       echo "### end stderr"
+    fi
+
+    echo "### start stdout"
+    cat stdout
+    echo "### end stdout"
+
+    echo "######### DONE WITH $1 #########"
+
+    if $FAIL; then
+      return 1
+    else
+      return 0
+    fi
+}
+
+# We aren't trying to catch errors here, this just let's us display the output
+archive_work() {
+    rm -f /tmp/sosreport*md5
+    mkdir /tmp/sosreport_test/
+    tar xfa /tmp/sosreport*.tar* -C /tmp/sosreport_test --strip-components=1
```

Comment:
Will this work also for a test case with `--build`, where the `sosreport` execution output is unpacked directory ("only")?

As this procedure is called after `run_expecting_success` one, everytime, doesnt it make sense to merge the procedures (i.e. print the plugin errors after `stdout/stderr`)?